### ISSUE TITLE
A brief clear title explaining the bug

### ISSUE THREAD

**Issue by craigdrown** [openmsupply/open-msupply#1937]
## What went wrong? 😲
Firefox: click the "Report" buttons and nothing happens

## Expected behaviour 🤔
Pretty report

## How to Reproduce 🔨
Use Firefox. Click button next to any report on "reports" list

## Your environment 🌱
Mac, Firefox 114,  online programs demo on 2023-06-26 (but it's been like that for ages)

**Comment by CarlosNZ** [comment]
> Click button next to any report on "reports" list

Do you mean the "Print" button? Cos I can't reproduce this, I get a Print dialog when I click it (on Firefox):

![Screenshot 2023-06-27 at 2 08 16 PM](https://github.com/openmsupply/open-msupply/assets/5456533/4acc97c7-100d-45cf-a34d-f21846ba183c)

**Comment by craigdrown** [comment]
Sorry @CarlosNZ  - updated title (yes, invoice forms work - just the reports in dispensary > reports > print)

**Comment by clemens-msupply** [comment]
The button works for me as well, maybe something wrong with the demo server or recent schema updates... Will need to do some more updates to the reports and then we can deploy an updated version. Will close this issue.

**Comment by nisha-dangol** [comment]
In Firefox, all Dispensary reports>Print button work for me, except for the `Viral Load Eligibility` one. I see the `Unable to print report` toast alert while printing it. Do I have to setup/add custom oms-report in cloud first for this by any chance? @clemens-msupply @roxy-dao 
<img width="1266" alt="image" src="https://github.com/openmsupply/open-msupply/assets/59193490/c7024580-e69d-4ab6-96ad-094c8c70c8ca">

**Comment by clemens-msupply** [comment]
Thank you for testing. There is another PR that affects the reports and the reporst already have been updated for these changes in a separat branch. I will let you know once everything is merged.
